crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/MeansAssessmentService.java

```diff
@@ -61,7 +61,9 @@ public ApiCreateMeansAssessmentResponse doAssessment(MeansAssessmentRequestDTO r
                     maatApiAssessmentResponse, assessmentCriteria, completedAssessment
             );
 
-            fullAssessmentAvailabilityService.processFullAssessmentAvailable(requestDTO, assessmentResponse);
+            if (requestType.equals(AssessmentRequestType.UPDATE)) {
```

Comment:
It might be worth switching around this statement to eliminate possible NullPointers.